src/app/pages/login/components/login-panel/login-panel.component.html

```diff
@@ -0,0 +1,82 @@
+<form class="flex flex-col gap-7 w-72" [formGroup]="loginForm" (ngSubmit)="submitForm()">
+  <span class="text-size-24 font-bold text-blue-600">{{ buildTranslationKeyForForm('login') | translate }}</span>
+
+  <div class="flex flex-col gap-5">
+    <div class="flex flex-col gap-2 h-24">
+      <app-text-input
+        [formControlName]="'email'"
+        [label]="buildTranslationKeyForPlaceholders('email') | translate"
+        [type]="'text'"
+        [placeholder]="buildTranslationKeyForPlaceholders('email') | translate"
+        [isValid]="emailControl.valid"
+        [isDirty]="emailControl.dirty && emailControl.touched"
+        [isPending]="emailControl.pending">
```

Comment:
I think we could use the isPending, because we are applying a specific style on the input element while its status is pending. This is used mostly for async validation. What do you think about this?